### DISCUSSION TITLE
Difference 7.x vs 20.x versions?

### DISCUSSION THREAD

**Discussion by renato-ll** [dbeaver/dbeaver#11890] [Q&A] [answered]
Hello friends

What are the Difference btw 7.x and 20.x versions?

**Comment by flambchap** [comment, marked as answer]
Hi. The main changes since 7.3:
- Big number of improvements in data transfer tool (insert or update, types mapping, ordering, etc)
- Spatial viewer: 3D objects support, custom tiles, SQLite support
- SQL editor panels (log, output, result tabs, toolbars) were significantly redesigned
- macOS: a lot of UI rendering bugs were fixed, BigSur support added
- A lot of database-specific issues were resolved and new features were added
- All packages now include Java (no need to install it separately)
- New databases: Trino, PostgreSQL for Google Cloud
- Product splash screen and icons were updated

To see all the changes we made, click [here](https://github.com/dbeaver/dbeaver/releases).
